handlers/content_modifier.go

```diff
@@ -0,0 +1,112 @@
+package handlers
+
+import (
+	"encoding/json"
+	"sync"
+
+	"github.com/layer5io/meshery/models"
+	"github.com/layer5io/meshery/models/pattern/core"
+	meshkube "github.com/layer5io/meshkit/utils/kubernetes"
+	"gopkg.in/yaml.v2"
+)
+
+//To be used while adding metadata to patterns,filters and applicationss
+type contentModifier struct {
+	token    string
+	provider models.Provider
+	prefObj  *models.Preference
+	userID   string
+}
+
+func newContentModifier(token string,
```

Comment:
What does "new" mean here? Can this be renamed?

Reply:
"new" prefixed function conventionally to create a new instance of a struct. And its not "New" because we dont want to export it

Reply:
Gotcha.